src/FVMAddress.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+pragma solidity ^0.8.30;
+
+import {RESOLVE_ADDRESS} from "./FVMPrecompiles.sol";
+
+library FVMAddress {
+    function resolveAddress(bytes memory filAddress) internal view returns (bool success, uint64 actorId) {
```

Comment:
I think we should be returning exists rather than success. success is uninteresting because it only fails if the address is invalid. should revert in that case.